package.json

```diff
@@ -52,11 +51,13 @@
   ],
   "homepage": "https://github.com/JPeer264/node-semantic-git-commit-cli#readme",
   "devDependencies": {
-    "ava": "^0.18.2",
-    "babel-cli": "^6.24.0",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-polyfill": "^6.23.0",
-    "babel-preset-env": "^1.2.1",
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/register": "^7.0.0",
+    "ava": "^1.0.0-rc.1",
```

Comment:
Why are you using a release candidate? Is there no support for babel 7 in a stable version?

Reply:
Unfortunately not. With the latest stable version (AVA 0.25.*) it did not work. But at least it is a rc

Reply:
If you want to work with rc it is fine for me too...